python/ray/serve/schema.py

```diff
@@ -514,17 +538,24 @@ class ServeApplicationSchema(BaseModel):
         description="Logging config for configuring serve application logs.",
     )
 
+    _allow_local_runtime_env_uris: bool = Field(
```

Comment:
Is this needed? I couldn't find what it is being used for.

Reply:
Oh! it's not, let me remove it. From a previous attempt at disabling the remote URI thing